patientIdentifierType.md

```diff
@@ -0,0 +1,44 @@
+# PatientIdentifierType
+
+## Overview
+Administrators define what types of identifiers they will collect. These range from National ID numbers, to driver's license numbers, to per-hospital medical record numbers.
+
+### Properties on PatientIdentifierType:
+
+* format: a regular expression defining what the identifier text should contain.
+
+* formatDescription: the text an admin can enter describing the regex format they just added
```

Comment:
An optional description of the regular expression (used to explain the requirements of the regular expression in terms a user would understand). For example, a regular expression like `\d{4,8}`could have a description like "Must be a number between 4 and 8 digits in length."